connector.go

```diff
@@ -53,17 +53,32 @@ func NewAWSReader(
 	ctx context.Context, accessKey string, secretKey string, regions []string, config *aws.Config,
 ) (AWSReader, error) {
 	var c = connector{}
+	const customEndpointRegion string = "custom"
```

Comment:
I meant to have a `raws.CustomRegion` basically, so it's something anyone using the lib can use to flag that instead of using a direct `"custom"` which is "worse".